compiler/src/dotty/tools/dotc/util/DiffUtil.scala

```diff
@@ -69,25 +65,21 @@ object DiffUtil {
     }
 
     val expectedDiff =
-      if (expected eq EOF) ANSI_EOF + expected + ANSI_DEFAULT
+      if (expected eq EOF) eof()
```

Comment:
You are printing less here (`expected` is not shown), is it intentional?

Reply:
The string for expected is in this case `"EOF"`. `EOF` is a string that has `"EOF"` but is also meant to be unique.